lib/bb/generate_report_request_form.rb

```diff
@@ -7,10 +7,10 @@ class GenerateReportRequestForm < Common::Form
                                 familyhealthhistory seifoodjournal healthcareproviders healthinsurance
                                 seiimmunizations labsandtests medicalevents medications militaryhealthhistory
                                 seimygoalscurrent seimygoalscompleted treatmentfacilities
-                                vitalsandreadings prescriptions medications vaallergies
+                                vitalsandreadings prescriptions vaallergies
                                 vaadmissionsanddischarges futureappointments pastappointments
                                 vademographics vaekg vaimmunizations vachemlabs vaprogressnotes
-                                vapathology vaproblemlist varadiology vahth wellness dodmilitaryservice ).freeze
+                                vapathology vaproblemlist varadiology vahth wellness dodmilitaryservice ).uniq.freeze
```

Comment:
Actually a cleaner approach would be to introduce a new coercer for UniqueArray that inherits from Virtus::Attributes::Array

```
class UniqueArray < Virtus::Attribute::Array
  def coerce
    super.uniq.compact
  end
end
```